upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -576,20 +576,24 @@ func (v *AutoscalingGroup) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Autos
 			changes.LoadBalancers = nil
 		}
 
-		var attachTGRequest *autoscaling.AttachLoadBalancerTargetGroupsInput
-		var detachTGRequest *autoscaling.DetachLoadBalancerTargetGroupsInput
+		var attachTGRequests []*autoscaling.AttachLoadBalancerTargetGroupsInput
+		var detachTGRequests []*autoscaling.DetachLoadBalancerTargetGroupsInput
 		if changes.TargetGroups != nil {
 			if e != nil && len(e.TargetGroups) > 0 {
-				attachTGRequest = &autoscaling.AttachLoadBalancerTargetGroupsInput{
-					AutoScalingGroupName: e.Name,
-					TargetGroupARNs:      e.AutoscalingTargetGroups(),
+				for _, tgsChunkToAttach := range sliceChunks(e.AutoscalingTargetGroups(), 10) {
```

Comment:
Maybe move 10 to a constant whose name explains why we're using that number?

Reply:
Should be better now.